lib/alephant/storage.rb

```diff
@@ -69,5 +70,14 @@ def get(id)
         :meta         => meta_data
       }
     end
+
+    private
+
+    def add_custom_meta(object)
+      {
+        :head_ETag            => object.etag,
+        :"head_Last-Modified" => Time.parse(object.last_modified).httpdate
```

Comment:
Semantically i'd prefer the use of `DateTime` as we're not just dealing with a time here - it's the date and the time.